python/cuml/svm/linear_svc.py

```diff
@@ -0,0 +1,192 @@
+            } (default = {LinearSVM_defaults.C})
+        The constant scaling factor of the loss term in the target formula
+          `F(X, y) = penalty(X) + C * loss(X, y)`.
+    grad_tol : {LinearSVM_defaults.grad_tol.__class__.__name__ \
+            } (default = {LinearSVM_defaults.grad_tol})
+        The threshold on the gradient for the underlying QN solver.
+    change_tol : {LinearSVM_defaults.change_tol.__class__.__name__ \
+            } (default = {LinearSVM_defaults.change_tol})
+        The threshold on the function change for the underlying QN solver.
+    tol : Optional[float] (default = None)
+        Tolerance for the stopping criterion.
+        This is a helper transient parameter that, when present, sets both
+        `grad_tol` and `change_tol` to the same value. When any of the two
+        `***_tol` parameters are passed as well, they take the precedence.
+    probability: {LinearSVM_defaults.probability.__class__.__name__ \
+            } (default = {LinearSVM_defaults.probability})
+        Enable or disable probability estimates.
+    multiclass_strategy : {{currently, only 'ovr'}} (default = 'ovr')
+        Multiclass classification strategy. ``'ovo'`` uses `OneVsOneClassifier
+        <https://scikit-learn.org/stable/modules/generated/sklearn.multiclass.OneVsOneClassifier.html>`_
+        while ``'ovr'`` selects `OneVsRestClassifier
+        <https://scikit-learn.org/stable/modules/generated/sklearn.multiclass.OneVsRestClassifier.html>`_
+    output_type : {{'input', 'cudf', 'cupy', 'numpy', 'numba'}} (default=None)
+        Variable to control output type of the results and attributes of
+        the estimator. If None, it'll inherit the output type set at the
+        module level, `cuml.global_settings.output_type`.
+        See :ref:`output-data-type-configuration` for more info.
+
+    Attributes
+    ----------
+    intercept_ : float, shape (n_classes,)
+        The constant in the decision function
+    coef_ : float, shape (n_classes, n_cols)
+        The vectors defining the hyperplanes that separate the classes.
+    classes_: float, shape (n_classes_,)
+        Array of class labels.
+    probScale_: float, shape (n_classes_ * 2,)
+        Probability calibration constants (for the probabolistic output).
+    n_classes_ : int
+        Number of classes
+
+    Notes
+    -----
+    The model uses the quasi-newton (QN) solver to find the solution in the
+    primal space. Thus, in contrast to generic :class:`SVC<cuml.svm.SVC>`
+    model, it does not compute the support coefficients/vectors.
+
+    Check the solver's documentation for more details
+    :class:`Quasi-Newton (L-BFGS/OWL-QN)<cuml.QN>`.
+
+    For additional docs, see `scikitlearn's LinearSVC
+    <https://scikit-learn.org/stable/modules/generated/sklearn.svm.LinearSVC.html>`_.
+    '''
+
+    REGISTERED_LOSSES = set([
+        'hinge',
+        'squared_hinge'])
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Sorry for the delay in response, I think we can merge as it is to 21.12 now that CI passed well and _if necessary_ we can revisit the docstring approach in a later release, the whole codebase could use a few improvements in that area so that could be part of that effort